src/ferc_xbrl_extractor/settings.py

```diff
@@ -0,0 +1,8 @@
+"""Define settings to use for extractor."""
+
+FERC1_OPTIONAL_COLUMNS = {
```

Comment:
Not sure if these should be constants, or configurable. And if we want to keep them as constants, I'm open to suggestions on where to place them.

Reply:
Each column might be nullable in some FactTables, but not others, right? It might be good to keep track of that, too - and then we can use that information when we're trying to call `Field.from_concept`.

Reply:
If we just want to set nullability by column name & not on a `(table, column)` basis, this location seems fine for me!